### DISCUSSION TITLE
800

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#975] [Comments]
# 800

silicon M1 맥북을 사용하고나서 IntelliJ를 계속 켜놓고 있으면 끝도 없이 메모리가 올라가는 이슈가 발생했다.현재 내 IntelliJ VM Option 에서 메모리 설정은 다음과 같다.(PC는 M1 Pro 16GB)-Xms1024m-Xmx4096m위와 같이 설정하고 IntelliJ를 켜두고 오랜 시간이 지나면 IntelliJ의 자체 Memory 분석 도구에서는 여전히 아래와 같이 정상적인 사용량이 노출되는 반면,실제 IntellIJ의 메모리 사용량은 다음과 같이 설정한 VM Option을 크게 초과해서 사용하는 것을 확인할 수 있다.16GB RAM을 가진 PC에서 14GB에 가까운 메모리를 사용중인데, 이건 더 높은 메모리 사양을 가진 맥북을 사용하면 더 크게 메모리를 점유한다.(테스트 결과..

https://jojoldu.tistory.com/800

**Comment by Hyeon9mak** [comment]
요즘 계속 골머리 아팠는데... 좋은 정보 공유 감사합니다!!

**Comment by joshua-qa** [comment]
해당 방법을 사용할 때 주의해야할 점이 있어서 적어봅니다 :)
jcef -> vanilla로 바꾸게 되면, `Leetcode Editor` 같은 플러그인에서 에러가 발생하게 됩니다.
-> jcef를 사용할 수 없어서 에러메시지가 발생하고, 해당 플러그인의 일부 기능이 정상 동작하지 않음

**Reply by daejoon** [reply]
Markdown의 Preview 기능도 동작하지 않습니다.

**Comment by qkrtkdwns3410** [comment]
메모리 10기가 -> 원래 설정한 4기가로 설정되네요 감사합니다..
